driver/src/test/java/org/neo4j/driver/integration/ErrorIT.java

```diff
@@ -52,7 +52,7 @@ public void shouldThrowHelpfulSyntaxError() throws Throwable
     }
 
     @Test
-    public void shouldNotAllowUsingTransactionAfterError() throws Throwable
+    public void shouldNotAllowUsingTransactionAfterRecoverableError() throws Throwable
```

Comment:
Well, currently we mark all `ClientError` as recoverable errors, which indicates the transaction is still open and a user could do something in the same transaction. The method name could be changed to anything for sure, such as `shouldNotAllowUsingTxAfterClientError`